### DISCUSSION TITLE
Say hello to the teams

### DISCUSSION THREAD

**Discussion by Jenwitsstt** [community/community#57171] [New to GitHub]
To entroduce myself to the member that I am quite very new in git hup community.Please be advice to Octocat also.

**Comment by queenofcorgis** [comment]
👋 Welcome to the GitHub Community on Discussions, @Jenwitsstt  Happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). This post will be removed within 24-48 hours to keep our community nice and tidy 🧹.

**Reply by Jenwitsstt** [reply]
Hello everyone
